modules/microshift-openshift-apis.adoc

```diff
@@ -16,3 +16,15 @@ In addition to the standard Kubernetes APIs, {product-title} includes a subset o
 | xref:../microshift_rest_api/security_apis/securitycontextconstraints-security-openshift-io-v1.adoc#securitycontextconstraints-security-openshift-io-v1[SecurityContextConstraints]
 | security.openshift.io/v1
 |===
+
+When an unsupported API is used via the {OCP} CLI tool (`oc`), an error message generates about a resource that cannot be found.
```

Comment:
```suggestion
When an unsupported API is used via the {OCP} CLI tool (`oc`), an error message may be printed about a resource that cannot be found.
```

The failure mode described here is most common, so I added "may be". I changed the verb because it felt not-quite-right, but if that's the style used elsewhere just ignore me.

Reply:
Needs to be present-tense, but some other wording wasn't great. PTAL at the next pass.